tools/appveyor.psm1

```diff
@@ -374,6 +374,12 @@ function Compress-CoverageArtifacts
     [System.IO.Compression.ZipFile]::CreateFromDirectory($resolvedPath, $zipOpenCoverPath) 
     $null = $artifacts.Add($zipOpenCoverPath)
 
+    $name = git describe
+
+    # Remove 'v' from version, prepend 'PowerShell' - to be consistent with other package names
+    $name = $name -replace 'v',''
+    $name = 'PowerShell_' + $name
+
```

Comment:
This code looks duplicate between `Compress-CoverageArtifacts` and `Invoke-AppveyorFinish`. Can we put it in a function and call that function instead?